pkg/Apps/Graph/Library/NodegraphGraph.js

```diff
@@ -5,10 +5,10 @@
  * license that can be found in the LICENSE file.
  */
 import {NodeCatalogNode} from '../../../Library/Node/NodeCatalogNode.js';
```

Comment:
I'm sorry I wasn't clear on the path forward.

I expect the entire Graph app is going to just go away in favor of GraphMaker, there is no reason to update these internal files.

Reply:
we should probably have a submitted `graph` file then, right?